### PR TITLE
Fix Menu Drawer overlapping whole Page

### PR DESCRIPTION
**Pull request by jomunker**
depends on https://github.com/vivid-planet/comet/pull/1137
depends on https://github.com/vivid-planet/comet/pull/1188

This PR fixes a bug where the menu drawer overlaps the whole page so nothing is clickable.
It's an MUI internal bug if variant is 'temporary' and the open state is initially true and gets toggled to false on the first render.

<details>
<summary>Screen recordings</summary>
<h3>Initial behaviour</h3>

https://github.com/vivid-planet/comet/assets/13380047/51aa767b-4879-423c-866f-fb3312478fe0

<h3>Fixed behaviour</h3>

https://github.com/vivid-planet/comet/assets/13380047/b65b53ad-ddd1-4b25-bd22-e12a71585e33

<h3>Behaviour of large screens after fix</h3>

https://github.com/vivid-planet/comet/assets/13380047/f2872131-fc7a-4000-ad93-e47d68639d47

</details>

### CODE REVIEW

**Comment by jomunker** [comment]
You mean if we update the MUI packages to a newer version? 

Don't we use the branch we branched away from as target branch if the PR isn't merged yet and the branch still exists?

**Comment by thomasdax98** [comment]
@johnnyomair No this wasn't fixed by https://github.com/mui/material-ui/pull/35010. To the contrary, the issue was introduced by this PR. If you revert the changes manually, our menu works as expected.

**Comment by thomasdax98** [comment]
@jomunker We always target either main or next. Otherwise it might be confusing which changes are for the current vs the next version.

You can mark the PR you are building on by putting 

> depends on https://github.com/vivid-planet/comet/pull/1137

at the top of the PR description.

**Comment by johnnyomair** [comment]
> @johnnyomair No this wasn't fixed by [mui/material-ui#35010](https://github.com/mui/material-ui/pull/35010). To the contrary, the issue was introduced by this PR. If you revert the changes manually, our menu works as expected.

Thanks! I didn't get that

**Comment by jomunker** [comment]
Alright thanks!

File: packages/admin/admin/src/mui/menu/Menu.tsx [hunk 1]
```diff
@@ -64,13 +67,14 @@ const MenuDrawer: React.FC<WithStyles<typeof styles> & MenuProps> = ({
             <Drawer
                 variant="temporary"
                 className={temporaryDrawerClasses.join(" ")}
-                open={temporaryOpen}
+                open={initialRender.current ? false : temporaryOpen}
```
Comment (thomasdax98) [inline packages/admin/admin/src/mui/menu/Menu.tsx:70]:
```suggestion
                // workaround for issue: https://github.com/mui/material-ui/issues/35793
                open={initialRender.current ? false : temporaryOpen}
```

Let's also put the hint here

**Review by thomasdax98 — APPROVED** [review]

**Pull request merged**